plugins/modules/homebrew_cask.py

```diff
@@ -79,7 +79,8 @@
     description:
     - Upgrade casks that auto update.
     - Passes --greedy to brew cask outdated when checking
-      if an installed cask has a newer version available.
+      if an installed cask has a newer version available,
+      or to brew cask upgrade when upgrading all casks.
```

Comment:
```suggestion
      or to C(brew cask upgrade) when upgrading all casks.
```

Reply:
Since C(brew cask) is deprecated, let's use the more modern syntax here:
```suggestion
      or to C(brew upgrade --cask) when upgrading all casks.
```